org.kde.iconexplorer.yml

```
@@ -0,0 +1,92 @@
+app-id: org.kde.iconexplorer
+runtime: org.kde.Platform
+runtime-version: 6.8
+sdk: org.kde.Sdk
+command: iconexplorer
+copy-icon: true
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
```

```suggestion
  - --socket=fallback-x11
  - --talk-name=org.freedesktop.FileManager1
```

it seems to open nautilus for me with this but then nautilus says "oops unable to handle this kind of location"

---

I think there's a bug in the application itself:
- with "Open with External Program", it uses e.g. `~/.local/share/flatpak/runtime/org.kde.Platform/x86_64/6.8/<checksum>/files/share/icons/breeze/apps/48/`
- with "Open containing folder" e.g. `/usr/share/icons/breeze/apps/48/`.
It works for me because I'm using a KDE system.

I will report this upstream.

---

https://invent.kde.org/plasma/plasma-sdk/-/issues/6

---

Per the comment, the app needs some upstream work to look into host icons https://docs.flatpak.org/en/latest/desktop-integration.html#icons

Not a blocker for me, but depends on what @flathub/kde thinks

---

@Justinzobel @travier can you answer this?

---

I'm on the side of just waiting a bit until the issue is resolved upstream, looks like it's pending action/review.